src/theme/cyber.js

```diff
@@ -1,8 +1,9 @@
 import { createTheme, responsiveFontSizes } from "@mui/material/styles";
 
-import Nunito from "../fonts/Nunito-Regular.ttf";
+import Nunito_Regular from "../fonts/Nunito-Regular.ttf";
 import Audiowide from "../fonts/Audiowide-Regular.ttf";
-
+import Nunito_Bold from "../fonts/Nunito-Black.ttf";
+import Nunito_Semi from "../fonts/Nunito-SemiBold.ttf";
```

Comment:
My only comment would be that we usually use PascalCase in react imports, i.e. no underscore: NunitoRegular, NunitoBold, etc...

There are cases where we use snake_case (or underscore_case) in js (e.g. in json comms with backend, e.g. legacy php backends), but they're quite rare, I mostly see camelCase for those.

Reply:
Oh and otherwise yeah the rest is great :p